Makefile

```diff
@@ -15,6 +16,9 @@ $(DOCKER_TARGETS): $(SUBDIRS)
 $(SUBDIRS):
 	$(MAKE) -C $@ $(MAKECMDGOALS)
 
+mvn_build:
```

Comment:
Should this be somewhere in `Makefile.maven` instead of here and be called `java_install` or something like that?

Reply:
OK, I'll move it.